.github/workflows/eden_emh_arm.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: EdenEMH_ARM
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+  workflows:
+    - Publish
+  types:
+    - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        conf_eden_server: ["-l dfw2 -c c2.large.arm"]
+        conf_eve_server: ["-l dfw2 -c c2.large.arm"]
+        hv: ["kvm"]
+    steps:
+      - name: Setup packages
+        run: |
+          sudo rm -rf /etc/apt/sources.list.d/microsoft-prod.list /etc/apt/sources.list.d/microsoft-prod.list.save
+          sudo add-apt-repository ppa:longsleep/golang-backports
+          sudo apt install -y golang jq expect
+      - name: Get eden
+        uses: actions/checkout@v2
+      - name: Create ubuntu server on packet hosting
+        id: ubuntu
+        run: |
+          packet_ubuntu_id=$(./shell-scripts/packet/create.sh ${{ matrix.conf_eden_server }} -p "$PACKET_PROJECT" -ns gh-action -os ubuntu_18_04)
+          packet_ubuntu_ip=$(./shell-scripts/packet/wait-eth0.sh $packet_ubuntu_id 100)
```

Comment:
Where is `shell-script` folder in eve? Did you forget to include new files in this PR?

Reply:
My misstake, fixed in the last commit.